ci/plan.yml

```diff
@@ -124,28 +143,43 @@ SUITES:
     timeout: 30
     conda_env: ci/conda/tensorflow.yml
     compare: no
-    cwd: plaidml/bridge/keras
     runner: python
     workloads:
       backend:
         args: [backend_test.py]
+        cwd: plaidml/bridge/keras
         conda_env: ci/conda/keras_backend_test.yml
         precision: untested
-      regression:
-        args: [regression_test.py]
+        skip_platforms:
+          - pml-vk-gen
+      edsl_cc_test:
+        runner: cc_test
+        cwd: tmp/input
         precision: untested
-        soft_fail: true
+        skip_platforms:
+          - pml-llvm-cpu
+        platform_overrides:
+          pml-vk-gen:
+            args:
+              - --plaidml_target=intel_gen
+              - --skip_test_file=edsl_test-skip.intel_gen.txt
```

Comment:
DRY?

Reply:
Where is this being repeated?

Reply:
The definition of `plaidml/edsl/tests:cc_test`

Reply:
The definition of `//plaidml/edsl/tests:cc_test`

Reply:
Well the issue here is that we're running this test outside of bazel, and so we can't rely on any bazel rules to run anything.

Reply:
Yes... but the previous code captured the arguments and propagated them to the out-of-bazel test, so that they'd only have to be defined in one place.

I don't know that it matters much, though.